From 1b005bbd1b34a77f3246ab4d227994849cf80a94 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Fri, 9 Sep 2016 14:30:01 +0900
Subject: [PATCH 1844/2848] HADOOP-13579. Fix source-level compatibility after
 HADOOP-11252. Contributed by Tsuyoshi Ozawa.

(cherry picked from commit 4ed74a917ce8da0a5ed0a72eb14de50155ecfd11)

Change-Id: I566f55587bd5f1bf66794cb1f9ed2d15c0348a21
---
 .../main/java/org/apache/hadoop/ipc/Client.java    |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java
index a948d43..c1cde1d 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Client.java
@@ -213,7 +213,8 @@ synchronized ExecutorService unrefAndCleanup() {
    * @param conf Configuration
    * @param pingInterval the ping interval
    */
-  static final void setPingInterval(Configuration conf, int pingInterval) {
+  public static final void setPingInterval(Configuration conf,
+      int pingInterval) {
     conf.setInt(CommonConfigurationKeys.IPC_PING_INTERVAL_KEY, pingInterval);
   }
 
@@ -224,7 +225,7 @@ static final void setPingInterval(Configuration conf, int pingInterval) {
    * @param conf Configuration
    * @return the ping interval
    */
-  static final int getPingInterval(Configuration conf) {
+  public static final int getPingInterval(Configuration conf) {
     return conf.getInt(CommonConfigurationKeys.IPC_PING_INTERVAL_KEY,
         CommonConfigurationKeys.IPC_PING_INTERVAL_DEFAULT);
   }
-- 
1.7.9.5

